docs/source/nlp/punctuation_and_capitalization.rst

```diff
@@ -0,0 +1,345 @@
+.. _punctuation_and_capitalization:
+
+Punctuation and Capitalization Model
+====================================
+
+Automatic Speech Recognition (ASR) systems typically generate text with no punctuation and capitalization of the words. \
+There are two issues with non-punctuated ASR output:
+
+- it could be difficult to read and understand;
+- models for some downstream tasks such as named entity recognition, machine translation or text-to-speech are usually trained on punctuated datasets and using raw ASR output as the input to these models could deteriorate their performance.
+
+Model Description
+-----------------
+
```

Comment:
A couple of sentences on the architecture and the fact it is based on BERT. Can we use docs from old nemo?